docs/controls/controls.rst

```diff
@@ -0,0 +1,54 @@
+=====================
+Qt Quick Controls 2.0
+=====================
+
+.. sectionauthor:: `e8johan <https://github.com/e8johan>`_
+
+.. github:: ch105
+
+.. note::
+
+    Last Build: |today|
+
+    The source code for this chapter can be found in the `assets folder <../../assets>`_.
+
+This chapter introduces how to use the Qt Quick Controls 2.0 to create a user interface built from standard components such as buttons, labels, sliders and so on. We will look at how various styles can be used to move user interfaces between platforms before diving into custom styling.
+    
+Introduction to Controls
+========================
+
+- What controls exist
+- How do the styles look
+- How to use them on desktop
```

Comment:
What kind of content did you have planned for this section?

Reply:
This is an outline. The contents would answer these questions and introduce the topic.

Reply:
No I mean for the "How to use them on desktop" section.

Reply:
Yes, I understand that it's an outline, I'm just curious about what you plan on putting here, on a high level.

Reply:
So the "implementation for desktop" part of the example will cover the "How to use them on desktop" section?